cmd/sidecars/sidecar_shim.go

```diff
@@ -195,6 +195,7 @@ func runPreCloudInitIso(vmiJSON []byte, cloudInitDataJSON []byte) ([]byte, error
 
 	log.Log.Infof("Executing %s", preCloudInitIsoBin)
 	command := exec.Command(preCloudInitIsoBin, args...)
+	command.Stderr = os.Stderr
 	return command.Output()
```

Comment:
Thank you @aep! Looks good!

Have you considered to return the combined output that includes stderr?
```go
return command.CombinedOutput()
```

Then we can log the output if an error occurs here:
https://github.com/kubevirt/kubevirt/blob/5ecef19996b770412a531bf2c46cbfadecc54a8a/cmd/sidecars/sidecar_shim.go#L110-L114

Something like:
```go
 	log.Log.Reason(err).Errorf("Failed OnDefineDomain: %s", string(newDomainXML)) 
```

I think it might be easier to fetch the logs that way, WDYT?

Reply:
Yup would work too.